### PR TITLE
Add DSTU 2 documentation for Encounter.

### PR DESCRIPTION
**Pull request by dayouJ**

### CODE REVIEW

File: lib/dstu2_resources.rb [hunk 1]
```diff
@@ -2663,5 +2663,168 @@ module Resources
           }
         ]
       }
+      DSTU2_ENCOUNTER_BUNDLE ||=
```
Comment (cerbs) [inline lib/dstu2_resources.rb]: Add a new line above this.

File: lib/dstu2_resources.rb [hunk 2]
```diff
@@ -2663,5 +2663,168 @@ module Resources
...
+                  "location": {
+                    "display": "Outpatient Ctr"
+                  }
+                }
+              ]
+            }
+          }
+        ]
+      }
+    DSTU2_ENCOUNTER ||=
```
Comment (cerbs) [inline lib/dstu2_resources.rb]: Add a new line above this.

**Comment by davischristi** [comment]
:+1:

File: lib/dstu2_resources.rb (hunk 1 above)
Comment (dayouJ) [inline lib/dstu2_resources.rb]: added.

File: lib/dstu2_resources.rb (hunk 2 above)
Comment (dayouJ) [inline lib/dstu2_resources.rb]: Added.

**Comment by cerbs** [comment]
:+1:

**Pull request merged**